src/plugins/log/log.js

```diff
@@ -19,7 +19,34 @@ const LEVEL_DISABLED = LEVEL_ERROR
 const COLORS = [DEBUG, INFO, WARN, ERROR, ERROR]
 const DESCRIPTIONS = ['debug', 'info', 'warn', 'error', 'disabled']
 
+/**
+ * @class Log
+ * @constructor
+ * @module plugins
+ * @example
+ * ### Using the Log
+ *
+ * you can use any of the 5 levels JAVASCRIPT:
+ * ```javascript
+ * // Disable Log
+ * Clappr.Log.setLevel(0);
+ * // Info Log
+ * Clappr.Log.setLevel(1);
+ * // Warn Log
+ * Clappr.Log.setLevel(2);
+ * // Error Log
+ * Clappr.Log.setLevel(3);
+ * // Debug Log
+ * Clappr.Log.setLevel(4);
+ * ```
```

Comment:
Please don't use the log levels as their numeric values. Instead, use their constants, which are defined as: `LEVEL_DEBUG`, `LEVEL_INFO`, `LEVEL_WARN` and `LEVEL_ERROR`. For some reason, the `LEVEL_DISABLED` wasn't exposed, so please add it as well.